lib/Collaboration/Collaborators/Listener.php

```diff
@@ -38,19 +38,13 @@
 use OCP\IUserManager;
 
 class Listener {
-
-	/** @var Manager */
-	protected $manager;
-	/** @var IUserManager */
-	protected $userManager;
-	/** @var Config */
-	protected $config;
+	protected Manager $manager;
+	protected IUserManager $userManager;
+	protected Config $config;
 	/** @var string[] */
-	protected $allowedGroupIds = [];
-	/** @var string */
-	protected $roomToken;
-	/** @var Room */
-	protected $room;
+	protected array $allowedGroupIds = [];
+	protected string $roomToken;
```

Comment:
```suggestion
	protected ?string $roomToken;
```
or
```suggestion
	protected string $roomToken = '';
```

or is it okay and will automatically initialise with empty string?